src/test/java/assertions/PersonTest.java

```diff
@@ -14,21 +16,37 @@
 
     @Test
     public void getFullNameReturnsFirstnameSpaceLastname(){
-        // TODO implement
-        throw new NotImplementedException();
+        Person p = new Person("Harry", "Moser", LocalDate.now());
+
+        String fullname = p.getFullName();
+
+        Assert.assertEquals("Harry Moser", fullname);
     }
 
     // TODO some more useful tests
+    @Test
+    public void getAgeReturnTestObObjektPeriodeZurckKommt(){
+        Person t = new Person("Harry", "Moser", LocalDate.now());
+        Period  period = t.getAge();
+        Assert.assertNotNull(period);
+
+    }
+    @Test
+    public void getAgeRetunObAlterRichtigRechnet() {
```

Comment:
Hi Harry
Cool! Danke für den PR. 
In einem Jahr würde dieser Test ja dann fehlschlagen. Was du machen kannst ist, ein Datum relativ zu Heute erzeugen: `LocalDate.now().minusYears(43);`. Wenn du dieses dem Konstruktor übergibst, wird der Test auch in einem Jahr noch laufen.
Aja, und dann kannst du auch die Testmethode etwas präziser formulieren ;)